source/aks/patching-artifactory.html.md.erb

```diff
@@ -23,130 +23,129 @@ If any issues arise during the update process, troubleshooting steps are provide
 
-• Click on "cft-ptlsbox-00-aks"
+  • Click on "cft-ptlsbox-00-aks"
 
-• Navigate to Connect and open Terminal
+  • Navigate to Connect and open Terminal
 
-• Connect using the provided commands (first three)
+  • Connect using the provided commands (first three)
 
-Note: Ensure correct cluster connection with the provided commands
+  * Note: Ensure correct cluster connection with the provided commands *
 
 3. Check Active Pods:
 
-• In the terminal, execute the following commands to display active pods:
+  • In the terminal, execute the following commands to display active pods:
 
-```command
-kubectl get pods -n artifactory 
-```
+  ```command
+  kubectl get pods -n artifactory 
+  ```
 
-![](Images/artifactory_pic02.png)
+  ![](Images/artifactory_pic02.png)
 
-This command lists running pods.
+  This command lists running pods.
 
-```command
-kubectl describe pods -n artifactory artifactory-oss-0 
-```
+  ```command
+  kubectl describe pods -n artifactory artifactory-oss-0 
+  ```
 
-![](Images/artifactory_pic03.png)
+  ![](Images/artifactory_pic03.png)
 
-This command provides detailed information about the desired pods, confirming the version (Image: 7.71.5)
+  This command provides detailed information about the desired pods, confirming the version (Image: 7.84.12)
 
 4. Version Checking:
 
-• Visit the following website to look for latest self hosted version:[Artifactory Release Information] (https://jfrog.com/help/r/jfrog-release-information/artifactory-7.77.5-self-hosted)
+  • Visit the following website to look for latest * self hosted version *: [Artifactory Release Information] (https://jfrog.com/help/r/jfrog-release-information/artifactory-release-notes)
 
 5. Pre-update Testing:
 
-• Before updating the PTLSBOX, conduct testing to ensure no issues
-
-• Download the repository from [GitHub] (https://github.com/hmcts/cnp-flux-config)
-
-• File location: apps/artifactory/artifactory/…
-
-• Create a new file named "artifactory-sbox.yaml" with the provided code snippet
-
-```command
-apiVersion: apps/v1
-kind: StatefulSet
-metadata:
-  name: artifactory-oss
-  namespace: artifactory
-spec:
-  serviceName: "artifactory"
-  replicas: 2
-  selector:
-    matchLabels:
-      app: artifactory-oss
-  template:
-    metadata:
-      labels:
-        app: artifactory-oss
-    spec:
-      containers:
-        - image: docker.bintray.io/jfrog/artifactory-oss:7.77.3
-          name: artifactory-oss
-```
-
-[Example PR] (https://github.com/hmcts/cnp-flux-config/pull/29609/files)
-
-6. Run Test:
-
-• Add the below code to kustomization.yaml file for the test to run:
+  • Before updating the PTLSBOX, conduct testing to ensure no issues
 
-```command
-patchesStrategicMerge:-../../artifactory/artifactory-sbox.yaml
-```
+  • Download the repository from [GitHub] (https://github.com/hmcts/cnp-flux-config)
 
-7. Push Changes:
-
-• Raise a PR and push changes out for review and approval
+  • File directory: apps/artifactory/artifactory/…
```

Comment:
Link to actual directory? 
Example: https://github.com/hmcts/cnp-flux-config/tree/master/apps/artifactory/artifactory

Reply:
yeah makes more sense